### DISCUSSION TITLE
Are there any new features planned for OpenSSL 3.3?

### DISCUSSION THREAD

**Discussion by fly2x** [openssl/openssl#22663] [Q&A]
We see that OpenSSL 3.2 will be released soon, and it brings a lot of new features, and I'm curious, will there be any new features in the next release?

**Comment by mattcaswell** [comment]
3.3 will likely include QUIC server support as well as performance enhancements. Likely also various updates related to FIPS. This list is roughly in priority order:

https://github.com/orgs/openssl/projects/2/views/28

**Reply by fly2x** [reply]
Great!

**Reply by mattcaswell** [reply]
Note: I should have said we will only tackle the top few items from that list in any one release. External contributors may contribute some from lower down the list (or that aren't on the list at all).
